### ISSUE TITLE
emitter leak

### ISSUE THREAD

**Issue by jonathanong** [TiddoLangerak/webpack-fail-plugin#1]
seems as though if i keep a webpack dev server running with this plugin, eventually it will have an event emitter leak.

```
(node) warning: possible EventEmitter memory leak detected. 11 beforeExit listeners added. Use emitter.setMaxListeners() to increase limit.
```

i suppose because it keeps attaching the same listener to the event: https://github.com/TiddoLangerak/webpack-fail-plugin/blob/cdf258dba4dada88ac7e226cd3dcf20ff2e58d34/index.js#L2

solution would be to add the event only if it hasn't been added to this plugin yet.

**Comment by TiddoLangerak** [comment]
Good idea! I'll also add detection for the dev server, because this plugin doesn't really make sense to be enabled when running the dev server.

**Comment by TiddoLangerak** [comment]
It should be fixed now in 1.0.4!
